python/semantic_kernel/utils/null_logger.py

```diff
@@ -35,6 +37,12 @@ class NullLogger(Logger, metaclass=_NullerMeta):
 
     def __init__(self):
         super().__init__(None)
+        logger.warning(
+            (
+                "NullLogger is deprecated and will be removed in a future release,",
+                "same as any 'log' and 'logger' arguments.",
```

Comment:
nit: "the same goes for all 'log' and 'logger' arguments."

Reply:
Fixed